apps/ledger-live-desktop/tests/page/account.page.ts

```diff
@@ -168,13 +168,18 @@ export class AccountPage extends AppPage {
     await this.seeGalleryButton.click();
   }
 
-  @step("Select NFT $0")
-  async selectNFT(nftName: string) {
-    await this.nft(nftName).click();
-  }
-
   @step("Navigate to NFT operation")
   async navigateToNFTOperation() {
     await this.nftOperation.click();
   }
+
+  @step("Expect NFT list to be visible")
+  async checkNftListInAccount(account: Account) {
+    if (account.nft) {
```

Comment:
I think we should check that account.nft exists, because if we forget to populate it, the test will skip the part silently